appeals/web/src/server/appeals/appeal-details/accordions/has.js

```diff
@@ -53,7 +52,24 @@ export function generateAccordion(appealDetails, mappedData, session, ipComments
 				}
 		  ];
 
-	const caseDocumentation = getCaseDocumentation(mappedData);
```

Comment:
What's the reasoning behind removing `case-documentation.js` and moving the code inline here and in the matching s78 mapper?

Reply:
This is the first piece of a lot of divergence that's going to happen between HAS and S78, there's lots more documentation for the latter